mycompany-seeq-link-connector-myconnector/build.gradle.kts

```diff
@@ -6,6 +6,9 @@ plugins {
 group = "com.mycompany.seeq.link.connector"
 version = "0.1.0-SNAPSHOT"
 
+project.version = "1.0.0.0"
+val minimumSeeqLinkSdkVersion = "100.0.0.0"
```

Comment:
I understand there is no released `100.0.x.x` version of the Link SDK in Java yet, but I think this CRAB won't be really complete until the minimum Link SDK declaration matches the version actually used as a dependency below. I think using different versions might be confusing for developers.

In the latest connector test framework there is a test called `testVersionDeclarations` that actually checks this.